README.md

```diff
@@ -38,114 +46,106 @@ The league establishes a map pool for each season. The map pool for Season 1 of
 
-### Divisions
+- Game 1: Home team selects map, Away team has first draft pick.
+- Game 2: Away team selects map, Home team has first draft pick.
+- Game 3: Home team selects map, Away team has first draft pick.
 
-Tournaments are separated by divisions. When a team joins a tournament, they will be joined by teams in the same division.
+The map selected must be from the [map pool](#map-pool) for the season. Each map may only be played once per series.
 
-### Seeding
-
-Tournament seeds are determined upon team rating. Tie-breakers are broken by who signed up first for the tournament.
+### Drafting
 
-### Rewards
+Use the in-game draft tool, set to tournament draft mode, for drafting.
 
-Rewards in tournaments unlock as more teams sign up. The rewards that can be earned from a tournament will differ based on the competitiveness, or division, of the tournament. More rewards will be given to Division 1 teams than Division 3 teams.
+### Results
 
-Only players who participated in a tournament are granted rewards for that tournament. Those players can [redeem their prizes in their account page](account_prizes_path).
+After the match finishes, either team can return to the match page and record the winner.
 
-## Matches
+If the match is recorded incorrectly, please contact paul@superjova.com to get it corrected.
 
-### Home Team
+## Seasons
 
-The Home team is responsible for hosting the match and inviting everyone.  If the match is being casted, that caster should be added to the lobby as an observer and set to "referee" status. 
+Seasons consist of 8 weeks of regular season play followed by playoffs for each division. Teams may continue to sign up at any point up until the end of the 6th week of play.
 
-### Map Selection
+### Standings
+      
+Division standings are determined by points, earned by winning regular season matches. Wins reward 3 points while losses reward no points.
 
-As each match is Best of 3 format, one team will have to select the initial battlefield for the two teams to clash.  The Home team, as designated by the schedule, will be responsible for selecting the first map of the match. The Away team will thus have the first pick in the draft. This then alternates each game, as such:
+### Roster Changes
 
-- Game 1: Home Team selects Map, Visiting Team has first Draft pick.
-- Game 2: Visiting Team selects Map, Home Team has first Draft pick.
-- Game 3: Home Team selects Map, Visiting Team has First Draft pick.
+Roster changes may be made up until the 6th week of regular season play. After that point, rosters will be locked and team registration will close.
 
-The map selected must be from the [map pool](#competitive-map-pool) for the season. Each map may only be played once per series. 
+### Playoff Seeding
 
-### Drafting
+Playoffs seeds are determined upon [team rating](#rating). Tie-breakers are broken by who signed up first for the tournament.
```

Comment:
Tie-breakers are broken should be Ties are broken.

The team that has completed more matches sounds better than who signed up first.